docs/output.md

```diff
@@ -2,56 +2,504 @@
 
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline. The directories listed below will be created in the results
+directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
-
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
+```text
```

Comment:
```suggestion
```tree
```

Reply:
- Change made in https://github.com/nf-core/oncoanalyser/pull/70